doc/source/index.rst

```diff
@@ -5,9 +5,17 @@ Embedding examples for `PyMechanical <_pymechanical_docs>`_ are contained in thi
 
 .. === EXAMPLES Gallery ===
 
+..
+   We have to include this rather than include it in a tree.
+
+.. include:: examples/index.rst
+   :start-line: 2
+
```

Comment:
Including this is will create a duplication of pages for basic/tech examples and an example section, is this expected?
![image](https://github.com/ansys/pymechanical-embedding-examples/assets/104772255/ae4978e5-1b07-42fa-a882-a135cb43c8a1)

Reply:
i might also change the repetition of "examples" in the heading , basic and technology show cases. as in pymapdl-examples https://examples.mapdl.docs.pyansys.com/index.html

Reply:
Fixed